model/Plan.js

```diff
@@ -8,8 +8,8 @@ const Schema = mongoose.Schema;
 const planSchema = new Schema({
   name: { type: String, required: true },
   majors: { type: [String] },
```

Comment:
please remove this line as major_id is the name of the major.

Reply:
hmm once we create major objects in db, don't they have ObjectIds separate from the `name` field?